packages/core/src/types.ts

```diff
@@ -79,7 +79,13 @@ export interface EditorOptions {
   };
   enableInputRules: EnableRules;
   enablePasteRules: EnableRules;
-  enableCoreExtensions: boolean;
+  /**
+   * If `false`, the core extensions will not be loaded.
+   * You can pass an object to disable specific extensions (e.g. `{ keymap: false }`).
+   * Extensions not listed in the object will still be enabled.
+   * @default true
+   */
+  enableCoreExtensions: boolean | Record<'editable' | 'clipboardTextSerializer' | 'commands' | 'focusEvents' | 'keymap' | 'tabindex', any>;
```

Comment:
```suggestion
  enableCoreExtensions: boolean | Record<'editable' | 'clipboardTextSerializer' | 'commands' | 'focusEvents' | 'keymap' | 'tabindex', false>;
```

Since `false` is the only valid option

Reply:
Oops, meant to put false.